core/ErrorHandler.php

```diff
@@ -161,8 +161,12 @@ public static function errorHandler($errno, $errstr, $errfile, $errline)
             case E_DEPRECATED:
             case E_USER_DEPRECATED:
             default:
+                $context = array('trace' => debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS));
```

Comment:
@katebutler could we add a limit here of say max 15 last traces? There is a second parameter `$limit` that can be set.